Podcast/FacebookFriendsTableViewCell.swift

```diff
@@ -12,6 +12,7 @@ protocol FacebookFriendsTableViewCellDelegate: class {
     func facebookFriendsTableViewCellDidPressFollowButton(tableViewCell: FacebookFriendsTableViewCell, collectionViewCell: FacebookFriendsCollectionViewCell, indexPath: IndexPath)
     func facebookFriendsTableViewCellDidSelectRowAt(tableViewCell: FacebookFriendsTableViewCell, collectionViewCell: FacebookFriendsCollectionViewCell, indexPath: IndexPath)
     func facebookFriendsTableViewCellDidPressSeeAllButton(tableViewCell: FacebookFriendsTableViewCell)
+    func facebookFriendsTableViewCellDidPressDismissButton(tableViewCell: FacebookFriendsTableViewCell, collectionViewCell: FacebookFriendsCollectionViewCell, indexPath: IndexPath)
```

Comment:
haha maybe change all these too if you get the chance. 
facebookFriendsTableView(didPress action: [NEW_ACTION_ENUM], on cell: CellType)
something like that. Can make the actions into an enum, makes it easy to add actions. 
Or at least rename and keep separate.